src/components/PostCard.jsx

```diff
@@ -70,8 +64,8 @@ const PostCard = ({ post, type }) => {
             />
           )}
           <img
-            className="w-full object-cover h-72 "
-            src={mediaURLs[activeMediaIndex]}
+            className="w-full object-contain h-72 "
```

Comment:
class naming can be better, which convention are you following?

Reply:
These are utility classes from tailwind